tests/unit_tests/test_wave_01.py

```diff
@@ -49,7 +49,10 @@ def test_removing_not_found_is_false():
 
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
+    #raise Exception("Complete this test according to comments below.")
+
+    assert result == False # making sure that False is returning  
```

Comment:
It is more "pythonic" style to do `assert not False` but this works just the same!